docs/guides/react-compiler.mdx

```diff
@@ -0,0 +1,31 @@
+---
+slug: guides/react-compiler
+title: Enabling React Compiler in Waku
+description: Learn how to enable the React Compiler in your Waku project using the Vite Plugin to optimize your React applications.
+---
+
+The React Compiler is a build-time tool designed to optimize React applications automatically. This guide will show you how to enable the React Compiler by integrating it with the `@vitejs/plugin-react`.
```

Comment:
I don't think it's build-time only.

Reply:
![image](https://github.com/user-attachments/assets/6f527915-0b5c-43de-b8cb-76fafa985999)

I copied it from React offical website

Reply:
Interesting. I wonder if raw Vite app + React Compiler works in DEV mode. I don't think this is Waku specific.

Reply:
dev mode is also a kind of build-time. built time in this context means it won't haveruntime cost, which will have no exta bundle size & cpu runs when you open browser

Reply:
I see.